src/libtrx/game/ui/dialogs/setting_tabs/gameplay_controls.def

```diff
@@ -26,9 +26,9 @@ X_UI_CFG_BOOL(gameplay.enable_tr2_swim_cancel,     GAMEPLAY_SETTINGS_ENABLE_TR2_
 X_UI_CFG_BOOL(gameplay.enable_wading,              GAMEPLAY_SETTINGS_ENABLE_WADING)
 X_UI_CFG_ENUM(gameplay.target_mode,                GAMEPLAY_SETTINGS_TARGET_MODE, .misc = UI_Settings_TargetModeEnumEntries)
 X_UI_CFG_BOOL(gameplay.enable_target_change,       GAMEPLAY_SETTINGS_ENABLE_TARGET_CHANGE)
-X_UI_CFG_BOOL(gameplay.enable_inverted_look,       GAMEPLAY_SETTINGS_ENABLE_INVERTED_LOOK)
 #endif
 
+X_UI_CFG_BOOL(gameplay.enable_inverted_look,       GAMEPLAY_SETTINGS_ENABLE_INVERTED_LOOK)
```

Comment:
Damn. It really was a good change, the x-macros for the UI settings I mean

Reply:
Agreed, adding to and tweaking the options is seamless 😌